services/configstore/types/types.go

```diff
@@ -355,3 +356,289 @@ type VariableValue struct {
 
 	When *types.When `json:"when,omitempty"`
 }
+
+type ActionType string
+
+const (
+	ActionTypeGetLog    ActionType = "getLog"
+	ActionTypeDeleteLog ActionType = "deleteLog"
+
+	ActionTypeGetRun     ActionType = "createRun"
```

Comment:
the value should be "getRun"

Reply:
@camandel Since these values are currently not persisted I just made them an int.